deployment/common/changeset/deploy_link_token.go

```diff
@@ -56,6 +58,40 @@ func DeployLinkToken(e deployment.Environment, chains []uint64) (deployment.Chan
 	return deployment.ChangesetOutput{AddressBook: newAddresses}, nil
 }
 
+// DeployStaticLinkToken deploys a static link token contract to the chain identified by the ChainSelector.
+func DeployStaticLinkToken(e deployment.Environment, chains []uint64) (deployment.ChangesetOutput, error) {
```

Comment:
Should this be unexported? Function sig does not exactly match that of a changeset.

Reply:
Nvm, thrown off by primitive type as "config"